packages/common/src/ui/layout/tables/components/Cells/CurrencyInputCell/CurrencyInputCell.tsx

```diff
@@ -15,7 +15,7 @@ export const CurrencyInputCell = <T extends RecordWithId>({
     Number(Number(column.accessor({ rowData, rows })))
   );
 
-  const updater = useDebounceCallback(column.setter, [rowData], 250);
+  const updater = useDebounceCallback(column.setter, [column.setter], 250);
```

Comment:
Row data is a param and so doesn't need to be in the deps array. Column.setter does, though!

without this, the setter was getting memoised and never updated (so if you were to 1) add an inbound line 2) update the first line - then the second line would just vanish